http_examples.cpp

```diff
@@ -67,7 +67,7 @@ int main() {
                 << name;
     }
     catch(const exception &e) {
-      *response << "HTTP/1.1 400 Bad Request\r\nContent-Length: " << strlen(e.what()) << "\r\n\r\n"
+      *response << "http/1.1 400 bad request\r\ncontent-length: " << strlen(e.what()) << "\r\n\r\n"
```

Comment:
The capital letters should be kept here?

Reply:
Yeah, they should be kept.
My mistake